src/main/java/org/opensearch/knn/index/query/nativelib/NativeEngineKnnVectorQuery.java

```diff
@@ -21,11 +21,7 @@
 import org.apache.lucene.util.Bits;
 import org.opensearch.common.StopWatch;
 import org.opensearch.knn.index.KNNSettings;
-import org.opensearch.knn.index.query.ExactSearcher;
-import org.opensearch.knn.index.query.KNNQuery;
-import org.opensearch.knn.index.query.KNNWeight;
-import org.opensearch.knn.index.query.PerLeafResult;
-import org.opensearch.knn.index.query.ResultUtil;
+import org.opensearch.knn.index.query.*;
```

Comment:
Pls avoid wildcard imports

Reply:
Sure. Somehow my IDE updated it to wildcard , will fix it. Thank you